### ISSUE TITLE
subscribeToStreamChanges returns invalid JSON

### ISSUE THREAD

**Issue by Llorx** [d-fischer/twitch#43]
## Bug Report

`subscribeToStreamChanges` returns `invalid json response body at https://api.twitch.tv/helix/webhooks/hub reason: Unexpected end of JSON input`

### Code

```ts
const id = "37768708"; // My user;
let listener = await WebHookListener.create(twitchClient, {port: 8081});
listener.listen();
listener.subscribeToStreamChanges(id, (stream) => {
    console.log(id, stream);
});
```

### Expected behavior

No error received.

### Actual Behavior

An error is thrown.

### Environment

- Version: 2.2.7
- Node version: v8.15.1
- Operating system: Windows 10

**Comment by d-fischer** [comment]
Hey! Thanks for the report. I just fixed this and pushed a new version 2.2.8 with that fix in it. Please try upgrading.

**Comment by Llorx** [comment]
So fast, nice.
